src/formatDate.js

```diff
@@ -9,6 +9,34 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const result = Array(3).fill(0);
+  const dateArray = date.split(fromFormat.slice(-1));
+  const day = dateArray[fromFormat.indexOf('DD')];
+  const month = dateArray[fromFormat.indexOf('MM')];
+  const yearFromFormat = fromFormat.indexOf('YY') === -1 ? 'YYYY' : 'YY';
+  const year = dateArray[fromFormat.indexOf(yearFromFormat)];
+  const yearToFormat = toFormat.indexOf('YY') === -1 ? 'YYYY' : 'YY';
+  const formatedYear = format(year, yearFromFormat, yearToFormat);
```

Comment:
The `format` function is used here but is not defined in the provided code. Ensure that this function is implemented elsewhere or provide its implementation to avoid runtime errors.